src/data/list/defs.lean

```diff
@@ -172,15 +172,6 @@ def lookmap (f : α → option α) : list α → list α
   | none   := a :: lookmap l
   end
 
-def map_with_index_core (f : ℕ → α → β) : ℕ → list α → list β
-| k []      := []
-| k (a::as) := f k a::(map_with_index_core (k+1) as)
-
-/-- Given a function `f : ℕ → α → β` and `as : list α`, `as = [a₀, a₁, ...]`, returns the list
-`[f 0 a₀, f 1 a₁, ...]`. -/
-def map_with_index (f : ℕ → α → β) (as : list α) : list β :=
```

Comment:
Why did this get removed?

Reply:
It's now in core and used for the widgets.